pkg/helpers/tokencmd/request_token_test.go

```diff
@@ -219,44 +219,23 @@ func TestRequestToken(t *testing.T) {
 				{initialHead, initialHeadResp},
 				{initialRequest, basicChallenge1},
 			},
-			ExpectedError: "challenger chose not to retry the request",
-		},
-
-		// Prompting basic handler
-		"prompting basic handler, no challenge, success": {
```

Comment:
I expected to keep the test and change the outcome

Reply:
this case will never happen, AFAICT, so took it out but maybe i'm missing something - i've added it back, with a failure instead.

Reply:
oops added this back, with `Handler: &BasicChallengeHandler{Username: "myuser", Reader: bytes.NewBufferString("mypassword\n")},` and successfulToken